cmd/build/v2/config.go

```diff
@@ -84,3 +86,13 @@ func (oc oktetoBuilderConfig) GetGitCommit() string {
 func (oc oktetoBuilderConfig) GetAnonymizedRepo() string {
 	return oc.repository.GetAnonymizedRepo()
 }
+
+func (oc oktetoBuilderConfig) GetBuildContextHash(buildInfo *model.BuildInfo) string {
```

Comment:
Could we add a test for this fn?

Reply:
2bd136b21e02efb9eaf6fdf36e4ef42173197240